src/components/datePicker/tests/DatePickerReducers.spec.ts

```diff
@@ -489,5 +489,94 @@ describe('Date picker', () => {
 
       expect(expectedState).toEqual(datePickerInitialState);
     });
+
+    it('should return the appliedLowerLimit if the lowerLimit is not defined', () => {
+      let oldState: IDatePickerState = _.extend({}, BASE_DATE_PICKER_STATE, {
+        lowerLimit: undefined,
+      });
+      let action: IReduxAction<IDatePickerPayload> = {
+        type: DatePickerActions.apply,
+        payload: {
+          id: 'some-date-picker',
+        },
+      };
```

Comment:
You could wrap the new tests in a before each and create a const for the action and another for the oldState